client/src/components/inspirationPage/inspiration/inspCard/inspCard.css

```diff
@@ -0,0 +1,30 @@
+
+@keyframes anim {
+  from {height:0px;}
+   to {height: 350px;}
+}
+.container-inspCardDetails{
+  display: grid;
+  grid-template-rows: 10% 85%;
```

Comment:
You should use fractions here instead to avoid problems with the box model. E.g. `1fr 4fr`. Check this for both css files.

Reply:
The image will be rounded so the border will not matter, and for now we have no pictures yet.
Aligning texted fixed